acl/src/main/java/org/apache/rocketmq/acl/plain/PlainPermissionManager.java

```diff
@@ -613,10 +615,11 @@ public void validate(PlainAccessResource plainAccessResource) {
 
         // Check the white addr for accessKey
         String aclFileName = accessKeyTable.get(plainAccessResource.getAccessKey());
-        PlainAccessResource ownedAccess = aclPlainAccessResourceMap.get(aclFileName).get(plainAccessResource.getAccessKey());
-        if (null == ownedAccess) {
+        if (aclPlainAccessResourceMap.get(aclFileName) == null ||
+                aclPlainAccessResourceMap.get(aclFileName).get(plainAccessResource.getAccessKey()) == null) {
```

Comment:
you can try getOrDefault() to reduce one time's get from map

Reply:
thanks for your suggestion, i will adjust it